website/docs/reference/widgets/input.md

```diff
@@ -24,6 +24,7 @@ These properties allow you to edit the Input widget. All of these properties are
 | **Error message**   | Sets the text of the error message to display if user input is considered invalid.                                                                                                                     |
 | **Placeholder**     | Sets the placeholder text within the input box. Use to show a hint or example value to the user.                                                                                                       |
 | **Tooltip**         | Sets a tooltip for the widget. You can add hints or extra information about the required input from the user.                                                                                          |
+| **Show step arrows**              | Control's visibility of step arrows for a number type input. When turned off: The step arrows to increment or decrement the values in the widget will be hidden.                                                                        |
```

Comment:
same as on the suggested change in the Currency input widget. Also, this is valid only for the Number input type right? so we should mention that.